From cd4a161350a97a42a07dc7f8acb3f4251c4ba11b Mon Sep 17 00:00:00 2001
From: Christian Boltz <apparmor@cboltz.de>
Date: Fri, 7 Aug 2020 22:37:19 +0200
Subject: [PATCH] Add CAP_BPF and CAP_PERFMON to severity.db

These capabilities were introduced in Linux 5.8

References: https://bugs.launchpad.net/bugs/1890547
(cherry picked from commit ae012502095596df4675555da635c868e3b3c04a)
Signed-off-by: John Johansen <john.johansen@canonical.com>
---
 utils/severity.db | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/utils/severity.db b/utils/severity.db
index 3c0284004..3e07d44e4 100644
--- a/utils/severity.db
+++ b/utils/severity.db
@@ -2,6 +2,7 @@
 #
 #    Copyright (C) 2002-2005 Novell/SUSE
 #    Copyright (C) 2014 Canonical Ltd.
+#    Copyright (C) 2020 Christian Boltz
 #
 #    This program is free software; you can redistribute it and/or
 #    modify it under the terms of version 2 of the GNU General Public
@@ -28,6 +29,7 @@
        CAP_SETGID 9
        CAP_SETUID 9
        CAP_FOWNER 9
+       CAP_BPF 9
 # Denial of service, bypass audit controls, information leak
        CAP_SYS_TIME 8
        CAP_NET_ADMIN 8
@@ -49,6 +51,7 @@
        CAP_BLOCK_SUSPEND 8
        CAP_DAC_READ_SEARCH 7
        CAP_AUDIT_READ 7
+       CAP_PERFMON 7
 # unused
        CAP_NET_BROADCAST 0
 
-- 
2.34.1

